community/sinit/build

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+make \
+    CFLAGS="$CFLAGS -static"
```

Comment:
Feel free to put this on one line ala `make DESTDIR="$1"`. :+1:

Reply:
ok